load.bzl

```diff
@@ -38,11 +38,11 @@ def repositories():
     if not native.existing_rule("bazel_toolchains"):
         http_archive(
             name = "bazel_toolchains",
-            sha256 = "6d54b26a58457f9fca2e54f053402061ffe73e3b909b8f6bf6dedb2a3db093ea",
-            strip_prefix = "bazel-toolchains-3.3.2",
+            sha256 = "882fecfc88d3dc528f5c5681d95d730e213e39099abff2e637688a91a9619395",
+            strip_prefix = "bazel-toolchains-3.4.1",
```

Comment:
Should this be `3.4.0`? https://github.com/bazelbuild/bazel-toolchains/releases/tag/3.4.0

Reply:
Thanks @hasheddan!
(That's the second release of theirs' I've seen recently w/ a misleading title.)

Reply:
@hasheddan -- Fixed!